docs/rancher/csi-driver.md

```diff
@@ -143,6 +143,34 @@ The only difference is in generating the `cloud-init` config where you need to s
 ./generate_addon_csi.sh <serviceaccount name> <namespace> k3s
 ```
 
+## Customize the default StorageClass
+
+Harvester CSI Driver provides the interface to pre-define the default StorageClass. If we do not specify the default StorageClass, the default StorageClass of Harvester CSI Driver will be the same as Harvester cluster.
+
+To customize the default StorageClass, you can use the parameter `host-storage-class`. Please check with the following example:
```

Comment:
```suggestion
The Harvester CSI driver provides the interface for defining the default StorageClass. If the default StorageClass in unspecified, the Harvester CSI driver and the Harvester cluster use the same default StorageClass. 

You can use the parameter `host-storage-class` to customize the default StorageClass.
```

Reply:
> Harvester CSI driver will use the default StorageClass of the host Harvester cluster.

Thanks for clarifying. The relationship was not very clear to me. We can change sentence #2 to this: 
"If the default StorageClass in unspecified, the Harvester CSI driver uses the default StorageClass of the host Harvester cluster."

Reply:
updated! Thanks!